program.md

```diff
@@ -61,5 +61,5 @@ css_id: program
 </section>
 
 <div class="md-cta-group">

```

Comment:
For code readability (as long as it doesn't degrade the rendering), put a newline between these HTML elements:

```
<a href="./community">Explore Community</a>
<a href="{{site.connect_url}}" target="_blank">Schedule Call</a>
```